x-pack/plugins/ml/server/routes/trained_models.ts

```diff
@@ -324,6 +325,40 @@ export function trainedModelsRoutes({ router, routeGuard }: RouteInitialization)
     })
   );
 
+  /**
+   * @apiGroup TrainedModels
+   *
+   * @api {post} /api/ml/trained_models/:modelId/deployment/_update Start trained model deployment
```

Comment:
Is this a copy and paste typo? Should it be `Update trained model deployment`?

Reply:
fixed in https://github.com/elastic/kibana/pull/144704/commits/56bcb60a0a4d2ea9ebd9cd6b19746096a66cb305